src/ducks/apps/components/ApplicationPage/Gallery.jsx

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from 'react'
+
+import { translate } from 'cozy-ui/react/I18n'
+import Modal, { ModalContent } from 'cozy-ui/react/Modal'
+
+export class Gallery extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      currentImage: null
+    }
+  }
+
+  onClick (imageUrl) {
+    this.setState({ currentImage: imageUrl })
+  }
+
+  onClose () {
+    this.setState({ currentImage: null })
+  }
+
+  render () {
+    const { t, slug, images } = this.props
+    const { currentImage } = this.state
+    if (!images) return null
+    const imagesParts = images.reduce((acc, image, index) => {
+      if (index < 2) {
+        if (!acc.big) acc.big = []
+        acc.big.push(image)
+      } else if (index < 5) {
+        if (!acc.small) acc.small = []
+        acc.small.push(image)
+      }
+      return acc
+    }, {})
+    return (
+      <div className='sto-app-images'>
+        {imagesParts.big && !!imagesParts.big.length &&
+          <div className='sto-app-big-images'>
+            {imagesParts.big.map((image, index) =>
+              <img
+                className='sto-app-big-image'
+                src={image}
+                alt={`${slug}-image-${index + 1}`}
+                onClick={() => this.onClick(image)}
```

Comment:
Could you add a `tabIndex=0` and a keydown event who trigger the click event when the key is 'Enter' to improve a11y ? Thx !

(I think we need at this point an a11y lib with this kind of snippet)